packages/renderer/src/lib/ui/AuditMessageBox.svelte

```diff
@@ -1,17 +1,17 @@
 <script lang="ts">
 import { faCircleInfo, faTriangleExclamation, faXmarkCircle } from '@fortawesome/free-solid-svg-icons';
-import type { AuditRecord, AuditResult } from '@podman-desktop/api';
+import type { AuditResult } from '@podman-desktop/api';
 import Fa from 'svelte-fa';
 
-export let auditResult: AuditResult;
+interface Props {
+  auditResult: AuditResult;
+}
 
-let infoRecords: AuditRecord[];
-let warningRecords: AuditRecord[];
-let errorRecords: AuditRecord[];
+const { auditResult }: Props = $props();
 
-$: infoRecords = auditResult?.records.filter(record => record.type === 'info');
-$: warningRecords = auditResult?.records.filter(record => record.type === 'warning');
-$: errorRecords = auditResult?.records.filter(record => record.type === 'error');
+const infoRecords = $derived(auditResult?.records.filter(record => record.type === 'info'));
+const warningRecords = $derived(auditResult?.records.filter(record => record.type === 'warning'));
+const errorRecords = $derived(auditResult?.records.filter(record => record.type === 'error'));
```

Comment:
`const` ?

Not sure this is a problem, but they seems to use `let` n the svelte documentation page https://svelte.dev/docs/svelte/$derived

(I know technically this is a proxy object, but might cause some issue ?)

Reply:
it's just a typescript definition.

if you never reassign the value on your (your = the code of the component) then you can use const.
I have the biomejs IDE plug-in reporting red underline when I use let

Svelte is able to change the value anyway as it'll intercept the values